### DISCUSSION TITLE
Is it possible to use Go1.21, for Cloudflare deployment?

### DISCUSSION THREAD

**Discussion by soyking** [gohugo-ananke/ananke#811] [2 - Q&A / Support] [answered]
I use [Cloudflare](https://developers.cloudflare.com/pages/framework-guides/deploy-a-hugo-site/) to build my site for long time. After upgrading to v2，I found that it requires Go1.22 but [Cloudflare's build image only support Go1.21](https://developers.cloudflare.com/pages/configuration/build-image/#supported-languages-and-tools). I know it's weird to adjust Go version for a specific content provider like Cloudflare, so I fork a new [branch](https://github.com/soyking/gohugo-theme-ananke/tree/v2.11.2_go_1.21) for integration. But I try to start this discussion since the Go version seems not the significant part in a template. Can it use Go1.21 at now?

**Comment by davidsneighbour** [comment, marked as answer]
You are right that the Go version is not influential on the workings of the theme. I will downgrade the version in `go.mod` in the next version as this topic came up more than once.

But: You are reading [that link](https://developers.cloudflare.com/pages/configuration/build-image/#supported-languages-and-tools) wrong. You can set `GO_VERSION` to `1.22.0` for your Cloudflare installation and all is well. I think that might help with newer features of GoHugo. While you are at it also define a newer `HUGO_VERSION` at the same space. [Here is how to override the values](https://developers.cloudflare.com/pages/configuration/build-image/#overriding-default-versions) of env variables.

**Reply by soyking** [reply]
Thanks for your reply. Sorry for my mistake, and I'll adjust go version in Cloudflare. So glad to use main branch again! ❤️
